app/modules/SilhouetteModule.scala

```diff
@@ -229,12 +231,7 @@ class SilhouetteModule extends AbstractModule with ScalaModule {
   /**
    * Provides the auth info repository.
    *
-   * @param totpInfoDAO The implementation of the delegable totp auth info DAO.
-   * @param passwordInfoDAO The implementation of the delegable password auth info DAO.
-   * @param oauth1InfoDAO The implementation of the delegable OAuth1 auth info DAO.
-   * @param oauth2InfoDAO The implementation of the delegable OAuth2 auth info DAO.
-   * @param openIDInfoDAO The implementation of the delegable OpenID auth info DAO.
-   * @return The auth info repository instance.
+   * XXX This should be done the same way as HTTP filters.
```

Comment:
So the theory here is that you should be able to define these from configuration, and have them loaded in by dependency injection automatically: see https://github.com/playframework/playframework/blob/master/core/play/src/main/scala/play/api/http/HttpFilters.scala#L78 for an implementation.